### PR TITLE
Update versions

### PR DESCRIPTION
**Pull request by dennys-bd**

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -1,8 +1,6 @@
 name: Tests
 on:
   push:
-    branches:
-      - master
```
Comment (fjsj) [inline .github/workflows/tests.yml:5]: I would say keep this just in case someone pushes to master.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -11,13 +9,7 @@ jobs:
...
     strategy:
       matrix:
-        include:
-          - python-version: 3.5
-          - python-version: 3.6
-          - python-version: 3.7
-          - python-version: 3.8
-          - python-version: 3.9
-          - python-version: "3.10"
+        python-version: ["3.5", "3.6", "3.7", "3.8", "3.9", "3.10", "3.11"]
```
Comment (fjsj) [inline .github/workflows/tests.yml:12]: Drop EoL versions if possible.

File: tox.ini [hunk 3]
```diff
@@ -11,9 +11,15 @@ envlist =
...
     # Django 4.0 supports Python 3.8, 3.9, and 3.10.
-    py{38,39,310}-django{40}
+    py{38,39,310}-django{400}
 
-    py38-djangolatest
+    # Django 4.1 supports Python 3.8, 3.9, 3.10, and 3.11.
+    py{38,39,310,311}-django{410}
+
+    # Django 4.2 supports Python 3.8, 3.9, 3.10, and 3.11.
+    py{38,39,310, 311}-django{420}
```
Comment (fjsj) [inline tox.ini:20]: Nitpick: Extra space after comma here.

**Review by fjsj — CHANGES_REQUESTED** [review]
Please check comments.

File: .github/workflows/tests.yml (hunk 1 above)
Comment (dennys-bd) [reply .github/workflows/tests.yml:5]: This is a restriction, in this case, it means only on master, so with it, branches don't run tests

**Review by fjsj — APPROVED** [review]

**Pull request merged**